src/sentry/static/sentry/app/views/performance/table.tsx

```diff
@@ -192,6 +220,12 @@ class Table extends React.Component<Props> {
       <EventsV2 eventView={eventView} organization={organization} location={location}>
         {({pageLinks, isLoading, tableData}) => (
           <div>
+            <StyledSearchBar
+              defaultQuery=""
```

Comment:
Should this reflect the current filtering query? For example the user does a filter and reloads. The transactions will be filtered but this search input won't be correct.

Reply:
I copied this component as is from Discover2 :) 

The `query` prop should handle this. Looks like `defaultQuery` has a default value. Matej has since converted thee searchbar component into TS.